code/__DEFINES/jobs.dm

```diff
@@ -140,3 +140,8 @@ MEDICAL_OFFICER, MEDICAL_RESEARCHER, SQUAD_LEADER, SQUAD_SPECIALIST, SQUAD_SMART
 #define SILO_ORIGIN "xenos from silo generation"
 
 #define SQUAD_MAX_POSITIONS(total_positions) CEILING(total_positions / length(SSjob.active_squads), 1)
+
+/// How many points a marine can spend in job specific vendors
+#define MARINE_TOTAL_BUY_POINTS		45
+/// How many points an engineer can spend 
+#define ENGINEER_TOTAL_BUY_POINTS 75
```

Comment:
30 more points is equivalent to the cost of all mats given for free in essential sets.